monkey/monkey_island/cc/ui/src/services/configuration/masquerade.js

```diff
@@ -0,0 +1,20 @@
+const MASQUERADE = {
+  'title': 'Masquerade',
+  'type': 'object',
+  'properties': {
+    'masque_list': {
+      'title': 'Masque list',
+      'type': 'array',
+      'uniqueItems': true,
+      'items': {'type': 'string'},
+      'default': [],
+      'description': 'List of masque strings that will be appended to the agent binaries '
```

Comment:
Don't say "appended". Anywhere. This is an implementation detail. Neither the user, the UI, or the API care at all about the specific implementation details and how this data is added to the binary.